fission-web-server/library/Fission/Web/Server/IPFS/Cluster/Error/Types.hs

```diff
@@ -0,0 +1,35 @@
+module Fission.Web.Server.IPFS.Cluster.Error.Types
+  ( ErrorBody (..)
+  , Error (..)
+  ) where
+
+import Fission.Prelude
+
+newtype ErrorBody = ErrorBody {message :: String}
+  deriving Show
+
+instance Display ErrorBody where
+  display = displayShow
+
+
+instance FromJSON ErrorBody where
+  parseJSON = withObject "ErrorBody" \obj -> do
+    message    <- obj .: "message"
+    return ErrorBody {..}
```

Comment:
Partly a question on this file, partly one on `Cluster.Error`: do we ever get an error response with more than `message` in it? If so, anything worth capturing in there? Any value in a `raw :: Text` field that just passes through the entire error??

Reply:
I don't believe so. The only errors I've seen just have a code and a message: id `{"code":400,"message":"error decoding Cid: selected encoding not supported"}`

We could just pass through the encoded JSON, but I don't see a huge benefit to it :man_shrugging: